Mage.Sets/src/mage/cards/a/AbaddonTheDespoiler.java

```diff
@@ -55,12 +50,16 @@ public AbaddonTheDespoiler(UUID ownerId, CardSetInfo setInfo) {
         this.addAbility(TrampleAbility.getInstance());
 
         // Mark of the Chaos Ascendant — During your turn, spells you cast from your hand with mana value X or less have cascade, where X is the total amount of life your opponents have lost this turn.
-        this.addAbility(new SimpleStaticAbility(new ConditionalContinuousEffect(
+        FilterSpell filter = baseFilter.copy();
```

Comment:
Stupid question 1: Why do you need to make a copy of the filter?

Reply:
The filter is static. So if we don't copy, we modify the filter for all AbaddonTheDespoiler cards. I saw another card do it (don't remember where off the top of my head)

We could make the filter not static, but then we have to add the CastFromZonePredicate(Zone.HAND) every time. It might  be cleaner that way? What is best practice?

Reply:
I am not sure that we're saving much computation by copying the filter rather than making it new each time.

The benefit of making it new would be that it can all be in one place so the file is less cluttered